client/app/translations/report-money-transfer.en.yml

```diff
@@ -57,6 +57,7 @@ errorPage:
   backToClientProfile: Back to client profile
 
 form:
+  thing: transfer
```

Comment:
"thing" is the noun used to describe the thing which you are going to add another of if you select the "yes" radio button for the add another field. It also generally represents the type of "thing" this form manages.